.github/workflows/push-foundation-repo.yml

```diff
@@ -1,17 +1,20 @@
+---
 name: Sync Cardinal-Cryptography repo with Aleph-Zero-Foundation repo
 
 on:
   push:
     branches:
       - main
-      - release-**
+      - 'release-*'
     tags:
-      - r-*
+      - 'r-*'
 
 jobs:
   sync-main:
     runs-on: ubuntu-20.04
-    if: github.repository == 'Cardinal-Cryptography/aleph-node' && startsWith(github.ref, 'refs/heads/') && github.ref_name == 'main'
+    if: >
```

Comment:
Let's make this `if` in a format of:
````
if: >
  something1(a, b, c)
  && something2(d, e, f)
  && something3(g, h, i)
````
where the conditions are not cut in half or something.  Whether `&&` is in the beginning or at the end - it doesn't matter I guess, pick one.

Reply:
done